src/middleware/auth/admin/adminAuthDataOnlyMiddleware.ts

```diff
@@ -12,18 +9,14 @@ const adminAuthDataOnlyMiddleware = async (req: Request, res: Response, next: Ne
 
   if (adminAccessToken == null) return res.sendStatus(401);
```

Comment:
To retrieve the token you can use https://www.npmjs.com/package/express-bearer-token